source/modules/assets/softwares.rst

```diff
@@ -0,0 +1,147 @@
+Softwares
+=========
+
+GLPI allows management of softwares and their versions as well as licenses, associated or not to software versions.
+
+A software is by default associated with an entity.
+
+.. ??? Un logiciel est par défaut associé à une entité : c'est-à-dire qu'il y aura autant de logiciels dans la base qu'il y a d'entités. Rendre un logiciel visible dans les sous-entités permet de faire une gestion plus fine.
+
+Financial management is done at the level of licenses; the financial management at software level is only a model for the licenses associated with this software.
+
+Softwares can be imported automatically using a third-party inventory tool; in this case a dictionary can be used to filter or clean the import data (see [Configure data dictionaries](07_Module_Administration/06_Dictionnaires.rst "The dictionaries are managed from menu entry Administration > Dictionaries")).
+
+Some fields are specific in the software form:
+
+- **Update** is an information, with no processing associated and which tells whether the software is an update of another software
+- **Category** allows to group softwares in the list of softwares of a computer
+- **Can be associated with a ticket** defines whether the software can be seen in the drop-down list "Hardware" of a  ticket
+
+.. ??? définit la visibilité du logiciel dans la liste déroulante "Matériel" d'un ticket.
+
+It is recommended to first create the software without a version number in the name, then to create the versions and last to create the licenses.
+
+***Note :*** in multi-entity mode, the list of software can rapidly become long because of double entries (one software per entity). A better approach can consist in grouping identical softwares in the same entity (see tab *Grouping* below), then to make recursive the elements that can be made recursive.
```

Comment:
```suggestion
.. note::
   In multi-entity mode, the list of software can rapidly become long because of double entries (one software per entity).
   A better approach can consist in grouping identical softwares in the same entity (see tab *Grouping* below), then to make recursive the elements that can be made recursive.
```